### PR TITLE
Restore page bottom margin

### PR DESCRIPTION
**Pull request by gtsiolis**
## Description

Following the changes in https://github.com/gitpod-io/gitpod/pull/7065, this will :a: revert the change from https://github.com/gitpod-io/gitpod/pull/4417 and :b: add some extra margin at the bottom of each page provides better UX when navigating long lists or large sections that reach the bottom of the viewport when scrolling to the end of the lists or sections.

## Related Issue(s)

Fixes https://github.com/gitpod-io/gitpod/issues/4634

### Screenshots

| BEFORE | AFTER |
|-|-|
| Screenshot 2021-12-30 at 12 00 02 AM (2) | Screenshot 2021-12-30 at 12 01 15 AM (2) |
| Screenshot 2021-12-29 at 11 59 52 PM (2) | Screenshot 2021-12-30 at 12 34 29 AM (2) |

## How to test

Try opening a few workspaces so that that list of worksspaces reaches the bottom of the viewport or going to user preferences on a viewport with a small height. Then, scroll down to the bottom of the page. Notice how this PR introduces some extra white space at the bottom of the page and lists or sections do not seem attached to the bottom of the viewport. 

## Release Notes

```release-note
Restore page bottom margin
```

### CODE REVIEW

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please ask for approval from **gtsiolis** after the PR has been reviewed.

Associated issue: *#4634*

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

<details open>
Needs approval from an approver in each of these files:

- **[components/dashboard/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/dashboard/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: components/dashboard/src/workspaces/Workspaces.tsx
Comment (gtsiolis) [inline components/dashboard/src/workspaces/Workspaces.tsx:170]: **thought**: Not sure why this didn't work with margin but using padding here does the trick.

**Review by gtsiolis — COMMENTED** [review]
@jankeromnes could you take a look at this small change when you are back?

**Review by jankeromnes — APPROVED** [review]
Looks good, and works as advertised! Much better this way 😌 ✨

/approve

**Comment by gtsiolis** [comment]
Thanks, @jankeromnes! 🐈

**Pull request merged**
